packages/mui-material/src/Slider/Slider.js

```diff
@@ -7,15 +7,15 @@ import {
   useSlotProps,
   unstable_composeClasses as composeClasses,
 } from '@mui/base';
-import { useSlider, getSliderUtilityClass } from '@mui/base/SliderUnstyled';
+import { useSlider } from '@mui/base/SliderUnstyled';
 import { alpha, lighten, darken } from '@mui/system';
 import useThemeProps from '../styles/useThemeProps';
 import styled, { slotShouldForwardProp } from '../styles/styled';
 import useTheme from '../styles/useTheme';
 import shouldSpreadAdditionalProps from '../utils/shouldSpreadAdditionalProps';
 import capitalize from '../utils/capitalize';
 import SliderValueLabelComponent from './SliderValueLabel';
```

Comment:
```suggestion
import SliderValueLabel from './SliderValueLabel';
```

we never use the `Component` suffix.

Reply:
I had to do this to avoid shadowing the name with the styled SliderValueLabel below. There's another way; I pushed a commit https://github.com/mui/material-ui/pull/35805/commits/fd4902d22ee7005c9ec658c0f94735be6dc2fe90 for it. Does it make sense?